docs/bash-completion

```diff
@@ -1,1101 +0,0 @@
-
-# Copyright 2016 The Kubernetes Authors.
```

Comment:
This PR is related to the issue here #1338 which is trying to remove this folder and use the --help contexts and README.md examples for feature exploration instead of /docs and it seems that not many users find information there and it keeping it up to date has caused issues.